NEWS.md

```diff
@@ -1,3 +1,72 @@
+# Sparklyr 1.3.0
+
+### Spark ML
+
+- `ml_evaluate()` methods are implemented for ML clustering and classification models
+
+### Distributed R
+
+- Created helper methods to integrate Spark SQL higher-order functions with
+  `dplyr::mutate`
+
+- Implemented option to pass partition index as a named parameter to `spark_apply()`
+  transform function
+
+- Enabled transform function of `spark_apply()` to return nested lists
+
+- Added option to return R objects instead of Spark data frame rows from transform
+  function of `spark_apply`
+
+- `sdf_collect()` now supports fetching Spark data frame row-by-row rather than
+  column-by-column, and fetching rows using iterator instead of collecting all
+  rows into memory
+
+### Connections
+
+- Hive integration can now be disabled by configuration in `spark_connect()` (#2465)
+
+- A JVM object reference counting bug affecting secondary Spark connections was fixed
+  (#2515)
+
+- Revised JObj envs initialization for Databricks connections (#2533)
+
+### Serialization
+
+- The new `arrow::write_to_raw()` functionality will be used if available in arrow-
+  enabled Spark connections
+
+- For arrow version 0.17 or above, sparklyr will ensure POSIXt columns have timezone
+  set before being serialized by arrow (columns without timezone will assume
+  `Sys.timezone()`)
+
+- Embedded nul bytes are removed from strings when reading strings from Spark to R
+  (#2250)
+
+- New `spark_read()` method to allow user-defined R functions to be run
+  on Spark workers to import data into a Spark data frame
+
+- `spark_write()` method is implemented allow user-defined functions to be run on
+  Spark workers to export data from a Spark data frame
+
+- Avro functionalities such as `spark_read_avro()`, `spark_write_avro()`,
+  `sdf_from_avro()`, and `sdf_to_avro()` are implemented and can be optionally
+  enabled with `spark_connect(..., package = "avro")`
+
```

Comment:
I think these ones fit better into `Data`, as in "Data Import/Export/Transformation"

```suggestion
### Data

- Created helper methods to integrate Spark SQL higher-order functions with
  `dplyr::mutate`
  
- New `spark_read()` method to allow user-defined R functions to be run
  on Spark workers to import data into a Spark data frame

- `spark_write()` method is implemented allow user-defined functions to be run on
  Spark workers to export data from a Spark data frame

- Avro functionalities such as `spark_read_avro()`, `spark_write_avro()`,
  `sdf_from_avro()`, and `sdf_to_avro()` are implemented and can be optionally
  enabled with `spark_connect(..., package = "avro")`

```